### PR TITLE
Use internal version pkg

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

**Comment by dajohi** [comment]
dcrstakepool version 1.4.0-pre+dev (Go version go1.11.5 appleIIc/amd64)

File: internal/version/version.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2013-2014 The btcsuite developers
+// Copyright (c) 2015-2018 The Decred developers
```
Comment (chappjc) [inline internal/version/version.go]: -2019

File: internal/version/version.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+const (
+	Major uint = 1
+	Minor uint = 4
+	Patch uint = 0
+)
+
+var (
+	// PreRelease is defined as a variable so it can be overridden during the
+	// build process with:
+	// '-ldflags "-X github.com/decred/dcrd/internal/version.PreRelease=foo"'
```
Comment (chappjc) [inline internal/version/version.go]: path says github.com/decred/dcrd

File: internal/version/version.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+	// if needed.  It MUST only contain characters from semanticAlphabet per
+	// the semantic versioning spec.
+	PreRelease = "pre"
+
+	// BuildMetadata is defined as a variable so it can be overridden during the
+	// build process with:
+	// '-ldflags "-X github.com/decred/dcrd/internal/version.BuildMetadata=foo"'
```
Comment (chappjc) [inline internal/version/version.go]: here too

File: internal/version/version.go [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+	// semanticBuildAlphabet defines the allowed characters for the build
+	// portion of a semantic version string.
+	semanticBuildAlphabet = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz-."
+)
+
+// These constants define the application version and follow the semantic
+// versioning 2.0.0 spec (http://semver.org/).
+const (
+	Major uint = 1
+	Minor uint = 4
```
Comment (chappjc) [inline internal/version/version.go]: Do we want to match dcrd's version?

**Pull request merged**